app/Metadata/GitHubFunctions.php

```diff
@@ -207,25 +207,6 @@ public function get_behind_text()
 		// @codeCoverageIgnoreEnd
 	}
 
-	/**
-	 * Return a string like 'commit number (branch)' or 'no git data found'.
-	 *
-	 * @return string
-	 */
-	public function get_info()
-	{
-		$branch = $this->get_current_branch();
-		$head = $this->get_current_commit();
-		if ($head == false || $branch == false) {
-			// when going through CI, .git exists...
-			// @codeCoverageIgnoreStart
-			return 'No git data found. Probably installed from release or could not read .git';
```

Comment:
Note mostly for myself - we don't need this because we have access to a release version now.